### PR TITLE
Bash completion support

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: tracer/resources/args_parser.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```
Comment (TingPing) [inline tracer/resources/args_parser.py]: Would be nice to be optional.

Comment (sean797) [inline tracer/resources/args_parser.py]: I disagree, even if we catch the `ImportError` what would be the point? `argcomplete` is going to get installed with every install.
I guess we could catch the `ImportError` and remove the dependency on `argcomplete` but if we do that we might as well not bother merging this..

Comment (TingPing) [inline tracer/resources/args_parser.py]: Note that this is packaged on other distros than Fedora where this package may not be available. There is no reason for it to be required.

Comment (sean797) [inline tracer/resources/args_parser.py]: @TingPing that makes sense! :+1: But what distros doesn't have argcomplete packaged?

Comment (TingPing) [inline tracer/resources/args_parser.py]: Arch doesn't have it in the official repos for example. While users can get it from the AUR it makes sense for it to be an optional dependency there.

Comment (sean797) [inline tracer/resources/args_parser.py]: Isn't Tracer in AUR also? I don't see the benefit of making it optional if the user doesn't have to enable an extra repo for it.

Comment (sean797) [inline tracer/resources/args_parser.py]: Okay, that makes more sense.. I wish you said that to start with! And I don't want you to have to apply patches every time you release. I wasn't tying to be obstructive, just trying to understand the use case.

I'll change the hard dependency to a soft one tomorrow :+1:  For arch maybe an `optdepends` would make sense

**Comment by FrostyX** [comment]
Thank you @sean797, it is awesome!

Can I ask you to change

    %if %{suggest}

to

    %if %{with suggest}

I think that it is a typo as you have it right on other lines :-)

**Comment by sean797** [comment]
Thanks for reviewing @FrostyX. Done!

**Pull request merged**